assignments/objects.js

```diff
@@ -19,27 +19,87 @@ const example = {
 
 // Write your intern objects here:
 
+const interns = [{
+  "id": 1,
+  "email": "mmelloy0@psu.edu",
+  "name": "Mitzi",
+  "gender": "F",
+},
+
+{
+  "id": 2,
+  "email": "kdiben1@tinypic.com",
+  "name": "Kennan",
+  "gender": "M",
+  "speak": function(){
+    console.log("Hello, my name is Kennan!");
```

Comment:
```suggestion
    return "Hello, my name is Kennan!";
```
You need to return the statement for it to be used